### PR TITLE
Header/Navigation animations

### PR DESCRIPTION
**Pull request by pavelkornev**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -35,9 +40,6 @@
     "gatsby-transformer-json": "^2.10.0",
     "gatsby-transformer-remark": "^2.16.1",
     "gatsby-transformer-sharp": "^2.12.1",
-    "gatsby-plugin-layout": "1.10.0",
```
Comment (evgenysegeda) [inline package.json:38]: break the build

File: src/components/Header/Navigation/index.js [hunk 2]
```diff
@@ -15,24 +28,98 @@ const query = graphql`
     }
 `;
 
-const Navigation = () => (
+const Navigation = ({ version, className = "" }) => {
```
[the remark this answers is not in the document]
Comment (pavelkornev) [reply src/components/Header/Navigation/index.js]: Remove

File: package.json (hunk 1 above)
Comment (pavelkornev) [reply package.json:38]: Fixed

File: src/components/Header/Navigation/index.js [hunk 3]
```diff
@@ -15,24 +29,105 @@ const query = graphql`
...
+  const onMouseEnter = useCallback((e) => {
+    calculateLineGeometry(e.target);
+  }, []);
+
+  const onMouseLeave = useCallback(() => {
+    calculateLineGeometry();
+  }, []);
+
+  const calculateLineGeometry = (activeElement) => {
+    console.log("geometry!");
```
Comment (evgenysegeda) [inline src/components/Header/Navigation/index.js:51]: Better to remove

File: src/components/Header/Navigation/index.js [hunk 4]
```diff
@@ -15,24 +29,105 @@ const query = graphql`
...
+      }
+
+      const lineStyle = getInlineStyles(lineDomNode);
+
+      if (activeElement) {
+        const activeElementGeometry = activeElement.getBoundingClientRect();
+        const lineWidth = activeElementGeometry.width - LINK_HORIZONTAL_PADDING;
+        const lineLeft = activeElementGeometry.left - parentLeftOffset + LINK_HORIZONTAL_PADDING / 2;
+
+        console.log("activeElementGeometry.left => ", activeElementGeometry.left);
```
Comment (evgenysegeda) [inline src/components/Header/Navigation/index.js:76]: the same

**Review by evgenysegeda — APPROVED** [review]

File: src/components/Header/Navigation/index.js (hunk 3 above)
Comment (pavelkornev) [reply src/components/Header/Navigation/index.js]: Done

File: src/components/Header/Navigation/index.js (hunk 4 above)
Comment (pavelkornev) [reply src/components/Header/Navigation/index.js]: Done

**Pull request merged**